site2/docs/deploy-docker.md

```diff
@@ -3,50 +3,97 @@ id: deploy-docker
+* Create broker container
 
-```shell
-docker run -it --name bookkeeper apachepulsar/pulsar-all:latest /bin/bash
-docker run -it --name zookeeper apachepulsar/pulsar-all:latest /bin/bash
-docker run -it --name broker apachepulsar/pulsar-all:latest /bin/bash
+```
+docker run -d --privileged=true -u=root -e PULSAR_PREFIX_metadataStoreUrl=zk:zookeeper:2181 -e PULSAR_PREFIX_cluster-name=cluster-a -e PULSAR_PREFIX_managedLedgerDefaultEnsembleSize=1 -e PULSAR_PREFIX_managedLedgerDefaultWriteQuorum=1 -e PULSAR_PREFIX_managedLedgerDefaultAckQuorum=1 --name broker apachepulsar/pulsar-all:latest /bin/bash
 ```
 
-### Create a network
-To deploy a Pulsar cluster on Docker, you need to create a `network` and connect the containers of ZooKeeper, BookKeeper and broker to this network. The following command creates the network `pulsar`:
+* Create bookie container
 
-```shell
+```
+docker run -d --privileged=true -u=root -e PULSAR_PREFIX_zkServers=zookeeper:2181 -e PULSAR_PREFIX_metadataServiceUri=metadata-store:zk:zookeeper:2181  --name bookie apachepulsar/pulsar-all:latest /bin/bash
+```
+## Create the network
+To deploy a Pulsar cluster on Docker, you need to create a network and connect the containers of ZooKeeper, BookKeeper and broker to this network. The following command creates the network pulsar:
+```
 docker network create pulsar
 ```
+Connect the containers of ZooKeeper, BookKeeper and broker to the pulsar network with the following commands.
 
-### Connect containers to network
-Connect the containers of ZooKeeper, BookKeeper and broker to the `pulsar` network with the following commands. 
-
-```shell
+```
 docker network connect pulsar zookeeper
+```
+```
 docker network connect pulsar bookkeeper
+```
+```
 docker network connect pulsar broker
+```
 
+## start the zookeeper, and init the cluster metadata
+Start the zookeeper service in the docker container by the following commands:
+1. Open the zookeeper container
+```
+docker exec -it zookeeper bash
+```
+2. Start the zookeeper service
+```
+bin/pulsar-daemon start zookeeper
+```
+3. init the cluster metadata
+```
+ bin/pulsar initialize-cluster-metadata \
+          --cluster cluster-a \
+          --zookeeper zookeeper:2181 \
+          --configuration-store zookeeper:2181 \
+          --web-service-url http://broker:8080 \
+          --broker-service-url pulsar://broker:6650 \
+```
+## start the broker and bookie service
+Start the broker and bookie service in the docker container by the following commands:
+### Start broker service
+1. Open the broker container
+```
+docker exec -it broker bash
+```
+2. Start the broker service
+```
+bin/pulsar-daemon start broker
+```
+
+### Start bookie service
+1. Open the bookie container
+```
+docker exec -it bookie bash
+```
+2. Start the bookie service
+```
+bin/pulsar-daemon start bookie
```

Comment:
Can you add an example of using a local pulsar client to produce and consume by connecting to the docker pulsar service?

Reply:
This is a deployment instead of ` get start`, right?
But I think we can export the ports.
And you can comment on the google doc.